clang/test/SemaSYCL/spec_const_accesors.cpp

```diff
@@ -0,0 +1,46 @@
+// RUN: %clangxx -fsycl-device-only %s -o %t.bc
+//
+//==----------- spec_const_accessors.cpp -----------------------------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// The test checks that Clang doesn't crash if multiple accessors are used
+// together with specialization constants in single kernel.
+
+#include <CL/sycl.hpp>
+
+using namespace sycl;
```

Comment:
Remove, since you are using another namespace `cl::sycl::` everywhere anyway?

Reply:
Done

Reply:
Done